app/assets/javascripts/discourse/app/components/sidebar/section-link-prefix.js

```diff
@@ -0,0 +1,25 @@
+import Component from "@glimmer/component";
+
+export default class extends Component {
+  get prefixValue() {
+    if (!this.args.prefixType && !this.args.prefixValue) {
+      return;
+    }
+
+    switch (this.args.prefixType) {
+      case "span":
+        return this.args.prefixValue
+          .reduce((acc, color) => {
+            if (color?.slice(0, 6)) {
```

Comment:
Can we have a more reliable way of detecting hex color? Maybe this function could be extracted into a lib?

https://github.com/discourse/discourse/blob/a65e0d566f66f55244f43512644b94a72cb39912/app/assets/javascripts/admin/addon/models/color-scheme-color.js#L99-L102

Reply:
Since hex colors can also just be 3 numbers e.g. `#000`

Reply:
Good call :+1: I've added the regexp for this.